efficientdet/third_party/graph_edit/edit.py

```diff
@@ -0,0 +1,223 @@
+# Copyright 2020 Google Research. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Various function for graph editing."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from third_party.graph_edit import reroute
+from third_party.graph_edit import select
+from third_party.graph_edit import subgraph
+from third_party.graph_edit import util
+from tensorflow.python.ops import array_ops as tf_array_ops
```

Comment:
It would be better to avoid importing tf internal stuffs?

For example, here you can simply import tensorflow as tf, and replace the tf_array_ops.placeholder as "tf.placeholder"?